src/features/distribute/components/Distributions.tsx

```diff
@@ -31,21 +33,37 @@ export function Distributions() {
 
   const payoutEventsByTransaction = data?.payoutEventsByTransaction;
   const distributionResult = data?.distributionResult.data;
-  const poolAmount = data?.poolAmount;
+  const poolAmount = data?.poolAmount?.data;
```

Comment:
as a suggestion, is there a way to have the useDistributeInfo return poolAmount in a different way to avoid using data.poolAmount.data?